### PR TITLE
Monaspace: v1.101

### LINKED ISSUE #1503: MonaspiceNe Nerd Font Mono ss06 causing characters to overlay on each other
#### 🗹 Requirements

- [x] I have searched the [issues](https://github.com/ryanoasis/nerd-fonts/issues) for my issue and found nothing related and/or helpful
- [x] I have searched the [FAQ](https://github.com/ryanoasis/nerd-fonts/wiki/FAQ-and-Troubleshooting) for help
- [x] I have searched the [Wiki](https://github.com/ryanoasis/nerd-fonts/wiki) for help

#### 🎯 Subject of the issue

Experienced behavior:

When enabling `ss06` on MonaspiceNe Nerd Font Mono, characters after consecutive `#`s end up overlayed with the `#`s:

image

I was able to confirm that MonaspiceNe Nerd Font and MonaspiceNe Nerd Font Propo have the correct behavior.

Screenshot is from vscode, but I was able to reproduce the issue in Windows Terminal as well:

image

Expected behavior:

The same behavior as Monaspace Neon/MonaspiceNe Nerd Font/MonaspiceNe Nerd Font Propo:

image

For comparison, with `ss06` disabled:

image

Example symbols:

The main symbol I've noticed affected is `#`. I haven't noticed any others, but as far as I know, `ss06` only affects `#`.

```
###$$$$
```

#### 🔧 Your Setup

- _Which font are you using (e.g. `Anonymice Powerline Nerd Font Complete.ttf`)?_
   - `MonaspiceNeNerdFontMono-*.otf`  - as far as I can tell, affects all variants of it. Confirmed with Light and Regular at least
- _Where did you get the file from (download link, self patched, source downloaded from link...)_
   - Releases page (v3.1.1) - `Monaspace.zip`, also tested `Monaspace.tar.xz`
- _Which terminal emulator are you using (e.g. `iterm2`, `urxvt`, `gnome`, `konsole`)?_
   - Windows Terminal, behavior also observed in vscode
- _Are you using OS X, Linux or Windows? And which specific version or distribution?_
   - Windows 11 Pro 22H2

#### ★ Screenshots (Optional)

image

image

`settings.json` for Windows Terminal:

image

Copyable settings:

```json
{
    "font": {
        "face": "MonaspiceNe Nerd Font Mono",
        "features": {
            "calt": 1,
            "liga": 1,
            "ss03": 1,
            "ss06": 1,
            "ss08": 1
        }
    }
}
```

### PR DESCRIPTION
**Pull request by Stealthii**
#### Description

This updates the source font files for Monaspace to the upstream v1.101 release, changes of which are described in the release notes.

https://github.com/githubnext/monaspace/releases/tag/v1.101

Fixes #1503, in addition to many issues registered upstream covered in the patch notes.

#### Requirements / Checklist

- [X] Read the [Contributing Guidelines](https://github.com/ryanoasis/nerd-fonts/blob/-/contributing.md)
- [x] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan.
- [X] If this contains a font/glyph add its origin as background info below (e.g. URL)
- [X] Verified the license of any newly added font, glyph, or glyph set. License is: SIL OPEN FONT LICENSE Version 1.1

### CODE REVIEW

**Comment by Stealthii** [comment]
An example of ligature fixes between `v1.000` and `v1.100`:

### v1.000

### v1.100

**Review by Finii — COMMENTED** [review]
This looks good, thanks for the nice PR!

Will merge directly before a release and re-update then when needed (i.e. if there is a 1.101 or something ;)

File: src/unpatched-fonts/Monaspace/update.sh [hunk 1]
```diff
@@ -15,9 +15,9 @@ typefaces=(Neon Argon Krypton Xenon Radon)
...
-    echo "Processing ${face} ..."
-    mkdir -p "${face}"
-    for weight in "${weights[@]}"; do
-        find "${TMP_DIR}" -name "Monaspace${face}-${weight}.otf" -exec mv "{}" "${face}" \;
-    done
+	echo "Processing ${face} ..."
+	mkdir -p "${face}"
+	for weight in "${weights[@]}"; do
+		find "${TMP_DIR}" -name "Monaspace${face}-${weight}.otf" -exec mv "{}" "${face}" \;
+	done
```
[the remark this answers is not in the document]
Comment (Stealthii) [reply src/unpatched-fonts/Monaspace/update.sh]: That was by complete accident, as there's no editorconfig for this project my global config took precedence. Ran a fixup and updated the PR branch, good catch!

**Comment by cjmaxik** [comment]
There was indeed version 1.101
https://github.com/githubnext/monaspace/releases/tag/v1.101

**Comment by Finii** [comment]
@cjmaxik Yes, thanks. As I said, I will re-update this PR just before Nerd Fonts' release.

**Comment by Finii** [comment]
Thanks again, also for keeping this current!

**Pull request merged**